netlink/logging.go

```diff
@@ -0,0 +1,30 @@
+package netlink
+
+import (
+	"bufio"
+	"io"
+	"log"
+
+	agentlog "github.com/DataDog/datadog-agent/pkg/util/log"
+)
+
+func getLogger() *log.Logger {
+	reader, writer := io.Pipe()
```

Comment:
Is this just the io pipe for this specific program?

Reply:
No. What this does is create an in-memory `Writer` that feeds into `reader`.  Go-conntrack takes a log.Logger which has a different interface than the agent logger.

Reply:
Should this return nil if we know the log level isn’t debug?